### PR TITLE
 [PLT-2181] Change wording for deactivating a user from "Make Inactive" to "Deactivate"

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
Change wording for deactivating a user from "Make Inactive" to "Deactivate"

#### Ticket Link
GitHub: https://github.com/mattermost/platform/issues/6075
Jira: https://mattermost.atlassian.net/browse/PLT-2181

#### Checklist
- [x] Includes text changes and localization file ([.../i18n/en.json]

### CODE REVIEW

**Comment by jasonblais** [comment]
Thanks for another contribution Carlos! There seem to be some issues generating a test server so I'll test this out early next week

**Comment by cpanato** [comment]
@jasonblais updated the string per your comment in pre-release

File: webapp/i18n/en.json [hunk 1]
```diff
@@ -2017,9 +2017,9 @@
   "team_members_dropdown.confirmDemotionCmd": "platform roles system_admin {username}",
   "team_members_dropdown.inactive": "Inactive",
   "team_members_dropdown.leave_team": "Remove From Team",
-  "team_members_dropdown.makeActive": "Make Active",
+  "team_members_dropdown.makeActive": "Activate",
```
Comment (jasonblais) [inline webapp/i18n/en.json]: do any of these new strings need to be changed in the corresponding .jsx or .go files?

Otherwise looks good

Comment (cpanato) [inline webapp/i18n/en.json]: done

**Pull request merged**